File: App/Minority/Minority/app/src/main/java/com/revich/mobile/minority/Usuariosxsala.java
package com.revich.mobile.minority;

/**
 * Created by Igal on 8/21/2017.
 */

public class Usuariosxsala {

    public int Id;
    public int Usuario;
    public int SalaDeJuego;
    public String NombreSalaDeJuego;
    public boolean Sigue;
    public boolean VotoEnBlanco;

    public void LlenarDatos(int usuario, int salaDeJuego)
    {
        this.Id=0;
        this.Usuario=usuario;
        this.NombreSalaDeJuego="";
        this.SalaDeJuego=salaDeJuego;
        this.Sigue=false;
        this.VotoEnBlanco=false;
    }
}

File: RestApi/RestApiMinority/RestApiMinority/Models/Usuariosxsala.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RestApiMinority.Models
{
    public class Usuariosxsala
    {
        public int Id { get; set; }
        public int Usuario { get; set; }
        public int SalaDeJuego { get; set; }
        public string NombreSalaDeJuego { get; set; }
        public bool Sigue { get; set; }
        public bool VotoEnBlanco { get; set; }
    }
}